cypress/support/parent-commands.ts

```diff
@@ -0,0 +1,168 @@
+     * @example <caption>Find div with ID "my-selector" from the main element.</caption>
+     * // returns Chainer<HTMLDivElement>
+     * cy.getInFixedContainer('#my-selector').then(...);
+     * @alias `get`
+     * @alias `invoke('innerHeight')`
+     * @alias `scrollIntoView({offset: {top: height, left: 0}})`
+     * @see https://on.cypress.io/guides/core-concepts/interacting-with-elements#scrolling
+     */
+    getInFixedContainer(selector: string): Chainable<Subject>;
+
+    /**
+     * Gets page header and asserts it it exists, and is visible with correct text.
+     *
+     * @param {string} headerText
+     * @alias `contains`
+     * @alias `should`
+     */
+    pageHeaderShouldBeVisible(headerText: string): void;
+
+    /**
+     * Asserts the overview table component renders correctly.
+     */
+    overviewTableRendersCorrectly(): void;
+
+    /**
+     * Asserts that:
+     * - anchored headings each have an `href` attribute that is prefixed by `#`
+     * - that only `h2` and `h3` headings have matching in-page navigation links.
+     */
+    checkInPageNavigationLinks(): void;
+
+    /**
+     * Asserts that:
+     * - anchored headings each have an `href` attribute that is prefixed by `#`
+     * - that only `h2` and `h3` headings have matching in-page navigation links.
+     */
+    checkPageAside(): void;
+
+    /**
+     * Asserts that:
+     * - at least one live preview is in page
+     */
+    checkLivePreviews(): void;
+
+    /**
+     * Asserts that:
+     * - at least one do/don't container is in page
+     */
+    checkDoDonts(): void;
+
+    /**
+     * Asserts that:
+     * - change log container is visible
+     * - change log header `h2` with text "Changlog" is visible
+     * - on click of header, the changelog content is visible.
+     */
+    checkChangelogRevealer(): void;
+  }
+}
+
+Cypress.Commands.add('getDocsPageContentArea', () => cy.get('#paste-docs-content-area'));
+
+Cypress.Commands.add('pageHeaderShouldBeVisible', (headerText) => {
+  cy.contains('h1', headerText).shouldBeVisible();
+});
+
+Cypress.Commands.add('overviewTableRendersCorrectly', () => {
+  cy.get('table').first().as('componentsTable').should('be.visible');
+  cy.get('@componentsTable').find('thead').find('tr').find('th').as('overviewTableHeaders');
+  cy.get('@componentsTable').find('tbody').find('tr').as('tableRows');
+
+  // Table has correct number of headers
+  cy.get('@overviewTableHeaders').should('have.length', 6);
+
+  // Table has correct text for headers
+  const headers = ['Name', 'Status', 'Code ready', 'Design assets', 'Documentation', 'Peer review'];
+  cy.get('@overviewTableHeaders').each((header, index) => {
+    // using include here to do a loose match of headers
+    cy.wrap(header).should('include.text', headers[index]);
+  });
+
+  // verify at least one row is rendered.
+  cy.get('@tableRows').its('length').should('be.greaterThan', 0);
+});
+
+Cypress.Commands.add('checkInPageNavigationLinks', () => {
```

Comment:
I'm not a big fan of globals in code and kind of wish these were regular, imported functions. I feel as if these global commands shouldn't have page specific content, and instead should be high level commands like checking the URL or messing with localstorage. JS things, not DOM things basically.

Reply:
This is a pretty standard pattern with cypress. These custom commands could also be functions, but I wrote them as custom commands since that's what was discussed in the kickoff. Happy to iterate on that, but for the purpose of these tickets that are in flight I'd prefer to stick to the current methods.